localstack/testing/pytest/marking.py

```diff
@@ -29,5 +36,10 @@ class Markers:
     parity = ParityMarkers
     snapshot = SnapshotMarkers
 
+    multiruntime: MultiRuntimeMarker = pytest.mark.multiruntime
```

Comment:
The multiruntime tests are kind of a generalization of the runtime markers such as `parametrize_python_runtimes` in `tests/integration/awslambda/test_lambda_runtimes.py:63`, right?

Reply:
They are actually a bit more. They're used in `test_lambda_common.py` and they go hand-in-hand with the injected `multiruntime_lambda` fixture which handles building the corresponding handler packages.

The code for the marker/fixture can be found in `tests/integration/awslambda/conftest.py`

Reply:
At some point we wanted to generalize this multiruntime lambda a bit to make it usable in other areas as well, but let's reevaluate this after we proceed with cdk-based test setups soon.